Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM gcr.io/google.com/cloudsdktool/cloud-sdk:383.0.1-alpine
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:412.0.0-alpine
 
 ENV KUBECTL_VERSION="v1.22.2"
```

Comment:
@afarid may be update kubectl as well. I think the other 2 vulnerabilities are because of kubectl

Reply:
indeed, I have upgraded to kubectl to 1.23.12